hw3/card.test.js

```diff
@@ -0,0 +1,49 @@
+//import task 2 
+const cardCreating = require("./card");
+
+let testCard = new cardCreating.Card(process.env.USERNAME,500);
+let testCardNormalName = new cardCreating.Card("IvanovIvan",500);
+let testCardWithoutValues = new cardCreating.Card();
+//tests
+test('check name on card with process.env',()=>{
+    expect(testCard.name).toBe(process.env.USERNAME);
+});
```

Comment:
Each test should be independent as possible. Now I see `testCard` as a dependency.

Reply:
Fixed